Address follow-up items for the Brainstorm audience demonstration (#309)
microsoft/FluidExamples: brainstorm/README.md

1. Navigate to `http://localhost:3000`

>   You'll be taken to a url similar to `http://localhost:3000/**#1621961220840**` the path `##1621961220840` specifies one brainstorm document.

Reviewer: ```suggestion
  You'll be taken to a url similar to `http://localhost:3000/**#1621961220840**` the path `#1621961220840` specifies one brainstorm document.
```

> 2. Create another chrome tab with `http://localhost:3000/**#1621961220840**`

Reviewer 2: ```suggestion
2. Navigate to the same url in another window or tab
```
Not really related to why this change is happening, but I'm not a fan of the existing wording

```
As shown above, a static prefix is attached to indicate which property this entry holds (`noteId`, `author`, `color`, etc), then to ensure the key is unique for each note, we attach the `noteId` after the static prefix. With this structure, we now ensured that properties for each note are stored individually.

<br />

| :warning: WARNING                                                                                                                                       |
|:--------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `SharedMap` does not store object in the DDS map the same way it would be stored in a conventional map data structure. It is recommended to only store the necessary primitive data types in `SharedMap`.

Reviewer 2: I don't think advising to only store primitive types is quite right - the problem is more that comparison isn't valid for other types.  Maybe something like this would be better?
```suggestion
| `SharedMap` does not preserve object references like a conventional map data structure, and object comparisons of `SharedMap` values will be invalid . In this case, it is recommended to only store the necessary primitive data types in `SharedMap` or implement a custom comparison function.
```

const authorInfo = audience.getMyself();
```

With `members` and `authorInfo` defined, we use these to achieve several tasks:

1. displaying all current active users
    * All current active users are displayed as FacePile, or a list of personas, on the top right corner of the app.   
2. displaying author name in persona tooltip

>     * When hovered over the note's persona, the author who created the name will be displayed dynamically.

Reviewer 2: ```suggestion
    * When hovering over the note's persona, the author who created the name will be displayed dynamically.
```
nit: matches tense of the parent point better

3. displaying likes for the note

>     * When hovered over the like button, a list of all the users that liked the note will be shown.

Reviewer 2: ```suggestion
    * When hovering over the like button, a list of all the users that liked the note will be shown.
```
same nit
